### DISCUSSION TITLE
Resource "update & continue editing"

### DISCUSSION THREAD

**Discussion by misakstvanu** [laravel/nova-issues#3669] [Q&A] [answered]
- Laravel Version: 8.74.0
- Nova Version: 3.30.0
- PHP Version: 8.0
- Database Driver & Version: -
- Operating System and Version: -
- Browser type and version: -
- Reproduction Repository: -

### Description:
When updating resource and using "update & continue editing" feature it is not possible to save multiple times. The app thinks the resource was updated meanwhile by a another instance since the page was loaded.
Error: "Another user has updated this resource since this page was loaded. Please refresh the page and try again."

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Create a resource using php artisan command and set up a text field
2. Open the resource in Nova and create new entry
3. Open the entry for editing
4. Make changes to the resource
5. Save the changes using "update & continue editing"
6. Make more changes
7. Save using any method -> error is thrown

**Comment by crynobone** [comment, marked as answer]
You can disable Traffic Cop feature if you have this issue: https://nova.laravel.com/docs/3.0/resources/#disabling-traffic-cop

And if you have suggestions to improve the Traffic Cop feature please suggest it under #3502
